### PR TITLE
Auto PR - New Resources: `azurerm_dev_center` / `azurerm_dev_center_project`

### PR DESCRIPTION
**Pull request by hc-github-team-tf-azure**
Regenerating the Terraform Provider based on the latest changes

### CODE REVIEW

File: internal/services/devcenter/registration_gen.go [hunk 1]
```diff
@@ -10,19 +10,23 @@ type autoRegistration struct {
...
-	return []sdk.Resource{}
+	return []sdk.Resource{
+		DevCenterProjectResource{},
+		DevCenterResource{},
+	}
 }
 
 func (autoRegistration) WebsiteCategories() []string {
 	return []string{
+		"Dev Center Project",
```
Comment (tombuildsstuff) [inline internal/services/devcenter/registration_gen.go:29]: This'll need fixing in Pandora, will PR this in the morning

**Review by tombuildsstuff — APPROVED** [review]

**Review by manicminer — CHANGES_REQUESTED** [review]
The acctests may need to run serially as the maximum number of Dev Centers seems to be 2 in total, perhaps 1 per region.

**Comment by tombuildsstuff** [comment]
> The acctests may need to run serially as the maximum number of Dev Centers seems to be 2 in total, perhaps 1 per region.

I was running this locally against `westeurope` provisioning 4 at a time yesterday, AFAIK the Quota is 5, but agreed on running this serially.

File: internal/services/devcenter/registration_gen.go (hunk 1 above)
Comment (tombuildsstuff) [reply internal/services/devcenter/registration_gen.go:29]: https://github.com/hashicorp/pandora/pull/3160

**Comment by tombuildsstuff** [comment]
Tests pass 👍 

There's a couple of additional changes in https://github.com/hashicorp/pandora/pull/3160 needed for this to ship, but this is fine to go for now.

**Pull request merged**